web-registry/src/types/amplitude.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@analytics/amplitude';
```

Comment:
this was a simple way to silence some warnings (i forget if they were warnings or errors). since we really only use a tiny portion of these plugin libraries (only during initialization of the plugins), i think it's ok to not have great typing.